src/components/Product/Specifications.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Heading, Table, Tbody, Tr, Td } from "@chakra-ui/react";
+
+import "../../styles/Specifications.css";
+
+export default function Specifications({ specifications }) {
+    return (
+        <div>
+            <Heading className="specification__heading" size="lg">Specifications</Heading>
+            {specifications.map((table) => {
+                return (
+                    <div className="specification__table">
+                        <Heading size="md">{Object.keys(table)[0]}</Heading>
+                        <Table variant="striped" colorScheme="teal">
```

Comment:
remove variant and colorScheme props

Reply:
![image](https://user-images.githubusercontent.com/30081376/138535900-8ef2516f-fd7d-41c6-a607-21ca88c48fb0.png) is what a table would look like if I remove the props.

How about 
![image](https://user-images.githubusercontent.com/30081376/138535916-559f5200-c0e8-4edc-8fd2-64c7998015ac.png)
where I remove the `colorScheme` prop?

Reply:
So remove only `colorScheme` and keep `variant`?